docs/config/index.md

```diff
@@ -81,14 +81,14 @@ All configuration options that are not supported inside a [workspace](/guide/wor
 ### include
 
 - **Type:** `string[]`
-- **Default:** `['**/*.{test,spec}.{js,mjs,cjs,ts,mts,cts,jsx,tsx}']`
+- **Default:** `['**/__tests__/**/*.?([cm])[jt]s?(x)', '**/?(*.){test,spec}.?([cm])[jt]s?(x)']`
 
 Files to include in the test run, using glob pattern.
```

Comment:
I'm not sure if the readability of these patterns is clear enough anymore. Maybe we should have some kind of description here exactly as Jest has? https://jestjs.io/docs/configuration#testmatch-arraystring

Just to describe what exactly is considered as test file. Especially the new special handling for `__tests__` directory where even `__tests__/test-utils.ts` would be considered as test.

Reply:
@sheremet-va I noticed you resolved this discussion. Do you think it's unnecessary? I was going to document it, just haven't had time yet.

Reply:
I thought it was implemented, sorry. We should definitely document it.